.github/integration/sda/config.yaml

```
@@ -86,8 +86,8 @@ oidc:
 server:
   cert: ""
   key: ""
-  # jwtpubkeypath: "/shared/keys/pub/"
-  jwtpubkeyurl: "http://localhost:8800/oidc/jwk"
+  jwtpubkeypath: "/shared/keys/pub/"
+  # jwtpubkeyurl: "http://localhost:8800/oidc/jwk"
```

```suggestion
  # jwtpubkeypath: "/shared/keys/pub/"
  jwtpubkeyurl: "http://localhost:8800/oidc/jwk"
```

Leave this as it is or the automated test will have issues.

---

I noticed it. Have reverted this commit.